src/main/java/org/reactome/release/resourcechecker/WebPageResourceChecker.java

```diff
@@ -0,0 +1,52 @@
+package org.reactome.release.resourcechecker;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
```

Comment:
Why is Selenium being used? Why not just use HttpClient?

Reply:
I tried using HttpClient, but it doesn't allow for loading content produced by JavaScript.  Selenium allows the page to load as it would in a browser and retrieve its content so it can be inspected.

Reply:
Hmm so are you looking for more than just a "200 OK" response? What other kinds of responses are you parsing?

Reply:
Yes, there will be code to check for particular error text or expected content.  Sometimes resources will return a "200 OK" response because the page loaded fine, but the actual content is letting you know there is an error.

Reply:
And that list of text messages will need to be maintained when these resources change their messages.

Reply:
Yes, but that should trigger a failure in checking the resource.  We can update it as they change and it would be a good warning since if the content changed, it might be a signal of something bigger we want to know about.